pylint_odoo/checkers/modules_odoo.py

```diff
@@ -364,11 +365,13 @@ def _get_odoo_module_imported(self, node):
                     odoo_module.append(packages[2])
         return odoo_module
 
+    @utils.check_messages('odoo-addons-relative-import')
```

Comment:
FYI modules_odoo is not using/processing this decorator (yet)

Reply:
Do you mean the decorators have no effect and the method will still run? Which means I should use `self.linter.is_message_enabled`?

Reply:
yes

Reply:
and so?